discovery/roles/postscripts/ubuntu/vars/main.yml

```diff
@@ -46,3 +47,7 @@ omnia_ubuntu_postscripts_path:
 # Usage: configure_roce.yml
 roce_postscripts_path:
   - { src: "{{ role_path }}/../ubuntu/templates/omnia_roce.j2", dest: "/install/postscripts/omnia_roce", mode: "755" }
+
+# Usage: configure_gaudi.yml
+gaudi_postscripts_path:
+  - { src: "{{ role_path }}/templates/omnia_gaudi.j2", dest: "/install/postscripts/omnia_gaudi", mode: "755" }
```

Comment:
@Aditya-DP  have also implemented in this way?

Reply:
@priti-parate we have made this change with with respect to gaudi postscript

Reply:
same as the previous comment